packages/o/openvpn3/monitoring.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
We don't normally put this in monitoring files.

Reply:
Maybe we should, but that's a different discussion. Removed!